scalyr_agent/monitor_utils/k8s.py

```diff
@@ -1474,6 +1518,16 @@ def query_api( self, path, pretty=0, return_temp_errors=False, rate_limited=Fals
                     raise K8sApiTemporaryError('Temporary error seen while accessing api: %s' % str(e))
                 else:
                     raise
+            finally:
+                # conditionally record latency regardless of exception
+                if log_responses:
```

Comment:
See previous comment regarding using a single boolean variable to represent these conditions.